theme-thl/thl.edn

```diff
@@ -0,0 +1,22 @@
+{:color1 "#519b2f"
+ :color2 "#29a0c1"
+ :color3 "#0060a6"
+ :color4 "#519b2f"
+ :info-bgcolor "#ffffff"
+ :danger-bgcolor "#e28b83"
+ :success-bgcolor "#cee798"
+ :warning-bgcolor "#e7d96f"
+ :info-color "#4d5a91"
+ :danger-color "#79302f"
+ :success-color "#3c763d"
+ :warning-color "#6f572f"
+ :logo-bgcolor "#ffffff"
+ :logo-name "thl-logo-fi.png"
+ :logo-name-sm "thl-logo-fi.png"
+ :logo-content-origin "content-box"
+ :phase-color "#eee"
+ :phase-color-active "#cad2e6"
+ :phase-color-completed "#519b2f"
+ :table-bgcolor "#d1edc4"
+ :table-stripe-color "#d1edc4"
```

Comment:
:table-bgcolor and :table-stripe-color are the same, so there are no stripes. Is this on purpose?

Reply:
Yes. I think @Macroz said we should get rid of the double colored stripes, this was the easiest way not touching Language bank.

Reply:
I think the stripes are not how modern sites do it. We should instead add a feature where on hover we show the row in highlight. It's much easier for the eyes with similar result.